src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -302,6 +357,23 @@ export class CdkMenuItemTrigger implements OnDestroy {
     }
   }
 
+  /**
+   * Subscribe to the overlays outside pointer events stream and handle closing out the stack if a
+   * click occurs outside the menus.
+   */
+  private _subscribeToOutsideClicks() {
+    if (this._overlayRef) {
+      this._overlayRef
+        .outsidePointerEvents()
+        .pipe(takeUntil(this._stopOutsideClicksListener))
+        .subscribe(event => {
+          if (event.target instanceof Element && !isClickInsideMenuOverlay(event.target)) {
```

Comment:
Is the `instanceof` check necessary because of something that might happen at run-time, or is it just to satisfy typescript that `event.target` is an element? If the latter, you can just do `isClickInsideMenuOverlay(event.target as Element)` since we know that something has to be an element to receive a click